### PR TITLE
1023/language drop down in native language

### PR DESCRIPTION
**Pull request by zhuochenwang**
I didn't want to change a lot fo the code but this is functional from my testing. I left a comment in the code for setBananI18n.js about whether or not the entire for loop that was used to translate locale names is necessary.

### CODE REVIEW

File: client/locations-list-map.js [hunk 1]
```diff
@@ -317,7 +317,16 @@ function multilineStringToNodes(input) {
 
 function createMakerMarkerContent(entry, separator) {
   // Text to go into InfoWindow
-  const contentTags = [ce('h5', separator ? 'separator' : null, ctn(entry.name))];
+  const contentTags = [];
```
Comment (nburt) [inline client/locations-list-map.js:320]: it looks like this is code from @ngottlieb previous pr, do you know how that got in here?

File: middleware/setBananaI18n.js [hunk 2]
```diff
@@ -61,11 +61,11 @@ module.exports = (req, res, next) => {
 
   const translatedLocales = [];
   let activeLocale;
-
+  // Do we still need the for loop?
```
Comment (nburt) [inline middleware/setBananaI18n.js:64]: we need to find the activeLocale still but don't need to translate it, we'd just need to also change the view that creates the locale dropdown

File: constants/locales.js [hunk 3]
```diff
@@ -2,33 +2,41 @@ module.exports = [
...
   },
   {
     i18nString: 'ftm-italian',
     localeCode: 'it',
+    nativeLang: 'Italiano',
   },
   {
     i18nString: 'ftm-polish',
     localeCode: 'pl',
+    nativeLang: 'Polski',
```
Comment (nburt) [inline constants/locales.js:25]: Are we using i18nString anymore? If not, we can remove from here + en.json

Comment (zhuochenwang) [reply constants/locales.js:25]: I don't think we are, I kept just in case

File: client/locations-list-map.js (hunk 1 above)
Comment (zhuochenwang) [reply client/locations-list-map.js:320]: Ahh I got completely lost in version control and had some old commits. I tried to commit the newest copy of location-list.js but there must have been some mixup. I'll revert it to the deleted line.

File: middleware/setBananaI18n.js (hunk 2 above)
Comment (zhuochenwang) [reply middleware/setBananaI18n.js:64]: Oh ok, so for loop stays to check for active locales but no more translation needed right? I'll change the code in a bit

File: client/locations-list-map.js (hunk 1 above)
Comment (zhuochenwang) [reply client/locations-list-map.js:320]: Yeah I checked the location-list.js and there are so many changes that I didn't intend to introduce. I only wanted to change setBanana and locale, do you know how I can fix this?

**Pull request closed without merging**